src/components/BookList.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Book from './Book';
```

Comment:
- this component is a "container" component and should be in the `src/containers/` folder